docs/src/main/asciidoc/spring-stream.adoc

```diff
@@ -75,13 +75,24 @@ spring.cloud.stream.gcp.pubsub.bindings.{CONSUMER_NAME}.consumer.ack-mode=AUTO_A
 If automatic resource creation is turned ON and the subscription and/or the topic do not exist for a consumer, a subscription and potentially a topic will be created.
```

Comment:
I think we should add a note here that, only anonymous subscriptions will be cleaned up.

Reply:
It's mentioned twice below this.